keras_hub/src/layers/preprocessing/image_converter_test.py

```diff
@@ -33,11 +35,21 @@ def test_unbatched(self):
         self.assertAllClose(outputs[:, :, 1], np.ones((4, 4)) * 0.301569)
         self.assertAllClose(outputs[:, :, 2], np.ones((4, 4)) * 0.852353)
 
-    def test_resize_batch(self):
+    @parameterized.parameters(
+        (True, False),
+        (False, True),
+    )
+    @pytest.mark.skipif(
+        backend.backend() == "torch",
```

Comment:
Just import `keras` above, and use `keras.config.backend()` here. Will fit with how we use this elsewhere in the library.

Reply:
Done